### PR TITLE
Upgrade Cypress

### PR DESCRIPTION
**Pull request by aruniiird**
Fixes: https://issues.redhat.com/browse/RHSTOR-6616

### CODE REVIEW

**Comment by aruniiird** [comment]
@alfonsomthd , @SanjalKatiyar , please take a look...

File: package.json [hunk 1]
```diff
@@ -148,12 +148,12 @@
...
     "comment-json": "4.x",
-    "cypress": "^10.10.0",
-    "cypress-multi-reporters": "^1.5.0",
+    "cypress": "^13.15.2",
+    "cypress-multi-reporters": "^2.0.4",
     "eslint": "^7.31.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-chai-friendly": "^0.7.2",
-    "eslint-plugin-cypress": "^2.12.1",
+    "eslint-plugin-cypress": "^4.1.0",
```
Comment (alfonsomthd) [inline package.json:156]: This change is already done in https://github.com/red-hat-storage/odf-console/pull/1657, so you'll have to rebase.

Comment (alfonsomthd) [reply package.json:156]: After rebase, this change will not be necessary.

**Comment by SanjalKatiyar** [comment]
@aruniiird plz add Jira story to the PR's description.

File: package.json (hunk 1 above)
Comment (aruniiird) [reply package.json]: Done

**Comment by alfonsomthd** [comment]
@aruniiird I see this error in E2E job:
>Error: Webpack Compilation Error
[tsl] ERROR in /go/src/github.com/red-hat-storage/odf-console/cypress/support.ts(26,17)
      TS2339: Property 'defaults' does not exist on type '{ debug(enabled: boolean, options?: Partial<DebugOptions>): void; }'.
    at handle (/go/src/github.com/red-hat-storage/odf-console/node_modules/@cypress/webpack-preprocessor/dist/index.js:212:23)
    at finalCallback (/go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:498:32)
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:576:17
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/HookWebpackError.js:67:2
    at Hook.eval [as callAsync] (eval at create (/go/src/github.com/red-hat-storage/odf-console/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/go/src/github.com/red-hat-storage/odf-console/node_modules/tapable/lib/Hook.js:18:14)
    at Cache.storeBuildDependencies (/go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Cache.js:126:37)
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:572:19
    at Hook.eval [as callAsync] (eval at create (/go/src/github.com/red-hat-storage/odf-console/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/go/src/github.com/red-hat-storage/odf-console/node_modules/tapable/lib/Hook.js:18:14)
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:568:23
    at Compiler.emitRecords (/go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:1044:4)
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:558:11
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:1007:14
    at Hook.eval [as callAsync] (eval at create (/go/src/github.com/red-hat-storage/odf-console/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/go/src/github.com/red-hat-storage/odf-console/node_modules/tapable/lib/Hook.js:18:14)
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:1004:27
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/neo-async/async.js:2818:7
    at done (/go/src/github.com/red-hat-storage/odf-console/node_modules/neo-async/async.js:3[522](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/red-hat-storage_odf-console/1659/pull-ci-red-hat-storage-odf-console-master-odf-console-e2e-aws/1854679963282706432#1:build-log.txt%3A522):9)
    at Hook.eval [as callAsync] (eval at create (/go/src/github.com/red-hat-storage/odf-console/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/go/src/github.com/red-hat-storage/odf-console/node_modules/tapable/lib/Hook.js:18:14)
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/lib/Compiler.js:833:33
    at /go/src/github.com/red-hat-storage/odf-console/node_modules/webpack/node_modules/graceful-fs/graceful-fs.js:143:16
    at /root/.cache/Cypress/13.15.2/Cypress/resources/app/node_modules/@packages/server/node_modules/graceful-fs/graceful-fs.js:143:16
    at /root/.cache/Cypress/13.15.2/Cypress/resources/app/node_modules/@packages/server/node_modules/graceful-fs/graceful-fs.js:143:16
    at /root/.cache/Cypress/13.15.2/Cypress/resources/app/node_modules/@packages/server/node_modules/graceful-fs/graceful-fs.js:61:14
    at FSReqCallback.oncomplete (node:fs:187:23)

File: cypress/cypress.config.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ export default defineConfig({
     setupNodeEvents,
     specPattern: 'cypress/tests/**/*.ts',
     supportFile: 'cypress/support.ts',
+    testIsolation: false,
```
Comment (alfonsomthd) [inline cypress/cypress.config.ts:10]: We should enable this (and refactor accordingly) in a follow-up PR.

Comment (alfonsomthd) [reply cypress/cypress.config.ts:10]: https://github.com/red-hat-storage/odf-console/pull/1675

**Pull request merged**